docs/core/install/linux-package-manager-ubuntu.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Install .NET Core on Ubuntu with APT - .NET Core
+description: Install .NET Core SDK and .NET Core runtime on Ubuntu using a package manager.
+author: thraka
+ms.author: adegeo
+ms.date: 05/04/2020
+---
+
+# Install .NET Core with a package manager on Ubuntu
+
+.NET Core is supported on different Ubuntu versions. This article describes how to install .NET Core on Ubuntu with APT. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version.
+
+| .NET Core | Ubuntu Version (LTS) | Ubuntu non-LTS Version |
+|-----------|----------------------|------------------------|
+| 2.1 (LTS) | 16.04, 18.04, 20.04  | 19.10                  |
+| 3.1 (LTS) | 16.04, 18.04, 20.04  | 19.10                  |
+
+.NET Core versions no longer supported and the last Ubuntu release it was supported with:
+
+| .NET Core | Ubuntu Version             |
+|-----------|----------------------------|
+| 3.0       | 16.04, 18.04, 18.10, 19.10 |
+| 2.2       | 16.04, 18.04, 18.10        |
+| 2.0       | 14.04, 16.04, 18.04        |
+
+## APT (Advanced Package Tool)
+
+Depending on which version of Ubuntu you're using, these commands are slightly different. The changes are usually to URI paths the command uses. You'll want to modify them to match your version of Ubuntu. These changes are called out with placeholders.
```

Comment:
I think this is asking way too much of our users--I would call it a regression.

Reply:
I agree. I wish I could use auto replacement some how, but it's a lot of work too to duplicate articles for a simple 5 character replacement in a URL...

Reply:
No idea what it takes, but is one of these more reasonable?

![image](https://user-images.githubusercontent.com/12819531/81334719-14ea7980-906c-11ea-9773-914b3d57f749.png)

Reply:
That actually is tougher. Unfortunately we have whitelists of tab names.... yeah I know... so we can't just use what makes sense, we have to file tickets and get names approved, and we would have to do that for every version of every distro.